packages/docs/integrations/expression.md

```
@@ -0,0 +1,58 @@
+
+`@rapiq/parser-expression` parses an expression language for filters — useful when a single string must carry a complex condition tree (search boxes, saved filters, CLI flags).
+
+```sh
+npm install @rapiq/core @rapiq/parser-simple @rapiq/parser-expression
+```
+
+## The language
+
+Conditions are function-call expressions; `and` / `or` / `not` compose them:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)